db/repos/transactions/index.js

```diff
@@ -127,7 +140,27 @@ class TransactionsRepository {
 	 * Transactions counter.
 	 */
 	countList(params) {
-		return this.db.one(Queries.countList, params, a => +a.count);
+		// Add dummy condition in case of blank to avoid conditional where clause
```

Comment:
Best is to wrap this whole thing into a function, like it was done in all other methods that generate queries.